### PR TITLE
add support for axis illustration files in push.py

### PR DESCRIPTION
**Pull request by zhaoxiong0211**

### CODE REVIEW

File: Lib/gftools/push.py [hunk 1]
```diff
@@ -98,11 +98,13 @@ def push_report(fp):
 def missing_paths(fp):
     paths = [fp.parent / p.path for p in parse_server_file(fp)]
     axis_files = [p for p in paths if p.name.endswith("textproto")]
+    axis_illustration_files = [p for p in paths if p.name.endswith("svg")]
```
Comment (m4rc1e) [inline Lib/gftools/push.py:101]: `endswith` can take a tuple, https://docs.python.org/3/library/stdtypes.html#str.endswith. This is cleaner:

```
axis_files = [p for p in paths if p.name.endswith(("textproto", "svg))]
```

Comment (zhaoxiong0211) [reply Lib/gftools/push.py]: thank you! resolved

**Comment by m4rc1e** [comment]
Apologies but I'm not aware of what you're working on. Are we adding illustrations for each axis? will they be visible on fonts.google.com?

**Comment by zhaoxiong0211** [comment]
> Apologies but I'm not aware of what you're working on. Are we adding illustrations for each axis? will they be visible on fonts.google.com?

sorry i should have linked this PR. https://github.com/google/fonts/pull/4470/files
We are aiming to add illustration to each axis and it shall be shown to users together with the description

**Comment by m4rc1e** [comment]
Ok cool. I'm happy with this pr.

We currently have two axis registries which is not ideal. One in google/fonts, the other over at https://github.com/googlefonts/axisregistry. @felipesanches recently made the latter. @zhaoxiong0211 would it be possible to also add your google/fonts pr to this repo? Ideally, we should drop the axis registry in google/fonts and just use the repo in the future.

**Comment by zhaoxiong0211** [comment]
PR in googlefonts/axisregistry
https://github.com/googlefonts/axisregistry/pull/29

**Review by nathan-williams — APPROVED** [review]

**Pull request merged**